x-pack/legacy/plugins/apm/public/hooks/useFetcher.integration.test.tsx

```diff
@@ -76,7 +76,8 @@ describe('when simulating race condition', () => {
 
   it('should render "Hello from Peter" after 200ms', async () => {
     jest.advanceTimersByTime(200);
-    await tick();
+
+    await wait();
```

Comment:
Nice 👍 Wouldn't mind if you removed `tick` entirely.

Reply:
Will do!

Reply:
This is fixed now.